### DISCUSSION TITLE
flux reconcile git repo error

### DISCUSSION THREAD

**Discussion by ejoseph27** [fluxcd/flux2#4737] [General]
Flux cli is not able to configure the gitops repo.
flux has been installed to a AKS cluster via helm release. When i execute any command on the gitops repo, it is not able to checkout and determine the repo. i think it is not able to parse the special characters in the url. What to do??
<img width="1909" alt="error" src="https://github.com/fluxcd/flux2/assets/49556699/ebebf727-74f3-47ad-8c89-a83806d52b9b">

**Comment by muandane** [comment]
Did you try to rename and remove the spaces and special characters ? Clis are iffy with unconventional naming schemes
